tests/update-ts-references.test.js

```diff
@@ -1,21 +1,26 @@
 const execSh = require('exec-sh').promise;
 const path = require('path');
+const fs = require('fs');
 
-const rootFolderYarn = path.join(process.cwd(), 'test-run/yarn-ws');
+const rootFolderYarn = path.join(process.cwd(), 'test-run', 'yarn-ws');
 const rootFolderYarnNohoist = path.join(
   process.cwd(),
-  'test-run/yarn-ws-nohoist'
+  'test-run', 'yarn-ws-nohoist'
 );
-const rootFolderPnpm = path.join(process.cwd(), 'test-run/pnpm');
-const rootFolderYarnCheck = path.join(process.cwd(), 'test-run/yarn-ws-check');
+const rootFolderPnpm = path.join(process.cwd(), 'test-run', 'pnpm');
+const rootFolderYarnCheck = path.join(process.cwd(), 'test-run', 'yarn-ws-check');
 const rootFolderYarnCheckNoChanges = path.join(
   process.cwd(),
-  'test-run/yarn-ws-check-no-changes'
+  'test-run', 'yarn-ws-check-no-changes'
 );
-const rootFolderLerna = path.join(process.cwd(), 'test-run/lerna');
+const rootFolderLerna = path.join(process.cwd(), 'test-run', 'lerna');
 const compilerOptions = { outDir: 'dist', rootDir: 'src' };
 
 const setup = async (rootFolder) => {
+  if (!fs.existsSync(rootFolder)) {
```

Comment:
`fs.exist` is [deprecated](https://nodejs.org/api/fs.html#fs_fs_exists_path_callback), since we are using Node 14, you might like to use https://nodejs.org/api/fs.html#fs_fspromises_stat_path_options as alternative.